pkg/cmd/root/root.go

```diff
@@ -53,6 +69,14 @@ func NewCmdRoot(f *cmdutil.Factory, version, buildDate string) *cobra.Command {
 		Annotations: map[string]string{
 			"versionInfo": versionCmd.Format(version, buildDate),
 		},
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
Moved from being added in `main.go`, it felt more appropriate to get set here where the root command is being created.